### ISSUE TITLE
Can't disable virtual output sometimes

### ISSUE THREAD

**Issue by colemickens** [swaywm/sway#5547]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions on IRC at #sway on irc.freenode.net. **acknowledged**
- Proprietary graphics drivers, including nvidia, are not supported. Please use the open source equivalents, such as nouveau, if you would like to use Sway. **acknowledged**
- Problems with the Wayland version of Firefox are likely to be Firefox bugs. Start by submitting your issue to the Firefox Bugzilla and come back here only after they confirm otherwise. **acknowledged**

### Please fill out the following:
```
 ~ ------------------------------------------------------------------------------------------- 16:25:57 
> swaymsg -t get_version           
sway version v1.5-rc2-39d677af (Jul 13 2020, branch 'master')

 ~ ------------------------------------------------------------------------------------------- 16:26:03 
> sway -v               
sway version v1.5-rc2-39d677af (Jul 13 2020, branch 'master')
```

- **Debug Log:**
  - incoming

- **Configuration File:**
This repros with the stock config.

- **Stack Trace:**
  - n/a

```
 ~ -------------------------------------------------------------------------------------- INT  16:24:02 
> swaymsg output HEADLESS-1 disable

 ~ ------------------------------------------------------------------------------------------- 16:24:10 
> swaymsg -t get_outputs           
Output HEADLESS-1 'headless headless '
  Current mode: 1920x1080 @ 60.000000 Hz
  Position: 3440,0
  Scale factor: 1.000000
  Scale filter: nearest
  Subpixel hinting: unknown
  Transform: normal
  Workspace: 10
  Max render time: off
  Adaptive sync: disabled
```

**Comment by RedSoxFan** [comment]
Duplicate of #5522
